docs/layers/js-tutorial.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Tutorial
+sidebar_label: Example in Javascript
+slug: /layers/js-tutorial
+---
+
+# Using a Layer in Javascript
+
+This example shows how to setup a Layer with multiple parameters and then run multiple experiments with different combinations of those parameters.
+
+## Create a Layer
+
+First, create a new layer by logging into the [Statsig Console](https://console.statsig.com), navigate to Experiments, select the Layers tab and then select Create New.
```

Comment:
layer is sometimes capitalized, sometimes not throughout this.  I think we decided not to capitalize these things (Anu can confirm)